Header header
float64 Schutzzeit
float64 Stopzeit
bool obsctacle_stop
